scripts/run_agent.sh

```diff
@@ -11,14 +66,21 @@ rm -Rf vargasjr_dev_agent-*
 if [ "$AGENT_ENVIRONMENT" = "preview" ] && [ -n "$PR_NUMBER" ]; then
         echo "Detected preview environment for PR $PR_NUMBER"
         
-        if [ -z "$GITHUB_TOKEN" ]; then
-            echo "Error: GITHUB_TOKEN not set for preview environment"
+        if [ -z "$GITHUB_PRIVATE_KEY" ]; then
+            echo "Error: GITHUB_PRIVATE_KEY not set for preview environment"
```

Comment:
Let's keep this simple for now. The create agent script should fetch the GitHub token by GitHub private key, then pass the GitHub token into this script